prusti-interface/src/lifetimes/lifetimes.rs

```diff
@@ -1,27 +1,32 @@
-use crate::environment::{
-    borrowck::facts::{AllInputFacts, AllOutputFacts, BorrowckFacts, Loan, PointIndex, Region},
-    mir_dump::graphviz::{loan_to_text, to_sorted_text},
+use crate::{
+    environment::{
+        borrowck::facts::{AllInputFacts, AllOutputFacts, BorrowckFacts, Loan, PointIndex, Region},
+        mir_dump::graphviz::{loan_to_text, to_sorted_text},
+    },
+    lifetimes::lifetime_formatter::{LifetimeString, opaque_lifetime_string},
 };
 use rustc_borrowck::consumers::{LocationTable, RichLocation};
 
+use rustc_middle::mir;
 use std::{
     cell::Ref,
     collections::{BTreeMap, BTreeSet},
     rc::Rc,
 };
+use crate::environment::mir_dump::graphviz::to_text::ToText;
 
 pub struct Lifetimes {
     facts: Rc<BorrowckFacts>,
     output_facts: AllOutputFacts,
 }
 
-pub(super) struct LifetimeWithInclusions {
+pub(crate) struct LifetimeWithInclusions {
     lifetime: Region,
     loan: Loan,
     included_in: Vec<Region>,
 }
 
-impl super::graphviz::ToText for super::lifetimes::LifetimeWithInclusions {
+impl ToText for super::lifetimes::LifetimeWithInclusions {
```

Comment:
I think that instead of pulling in the `ToText` trait it would be better to implement `std::fmt::Display`